watertap/unit_models/tests/test_osmotically_assisted_reverse_osmosis_0D.py

```diff
@@ -56,6 +56,7 @@
     MassTransferCoefficient,
     PressureChangeType,
 )
+from watertap.core.membrane_channel_base import SherwoodNumberEq
```

Comment:
Should we add the `SherwoodNumberEq` to watertap/core/__init__.py (like `PressureChangeType`)?